### PR TITLE
Braydonpublish

### PR DESCRIPTION
**Pull request by BraydonKains**
- Padding to buttons and made table fit 100% of container for style parity
- Search box with same functionality
- Mark Exempt/Unexempt and update exemption count responsively

Note: I noticed that there are "Braydon" and "Braydon Kains" commits, this is because I had my configuration set differently between my WSL and Powershell so when I committed with one or the other it does that.

### CODE REVIEW

File: polymer.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "npm": true,
-  "entrypoint": "d2l-activity-exemptions.js",
+  "entrypoint": "index.html",
```
Comment (jwong-d2l) [inline polymer.json]: setting this here will make it so when we actually load the app, it'll be pointed to the referenced demo page, so maybe put it back to the js file.

**Comment by benyamin-noori** [comment]
I see we have security alerts on this repo that can be fixed by bumping the versions of a few dependencies. 

Do you want to update the package lock here as well? Or would you rather open a separate PR? I'm fine with both!

File: package-lock.json [hunk 2]
```diff
@@ -8301,53 +9094,69 @@
       }
     },
     "mocha": {
-      "version": "5.2.0",
-      "resolved": "https://registry.npmjs.org/mocha/-/mocha-5.2.0.tgz",
-      "integrity": "sha512-2IUgKDhc3J7Uug+FxMXuqIyYzH7gJjXECKe/w43IGgQHTSj3InJi+yAA7T24L9bQMRKiUEHxEX37G5JpVUGLcQ==",
+      "version": "3.5.3",
```
Comment (jwong-d2l) [inline package-lock.json]: Not sure why this ended up getting downgraded.  Thoughts?

Comment (BraydonKains) [reply package-lock.json]: Not sure about this one; it's not a dependency we put into package.json, I believe it's a dependency of Polymer 3. At least it's the testing framework used. Not sure exactly the way to bump it back up.

**Comment by BraydonKains** [comment]
> 
> 
> I see we have security alerts on this repo that can be fixed by bumping the versions of a few dependencies.
> 
> Do you want to update the package lock here as well? Or would you rather open a separate PR? I'm fine with both!

Should I delete the package-lock and run npm audit? This is my first time actually fixing a situation like this.

File: package-lock.json (hunk 2 above)
Comment (jwong-d2l) [reply package-lock.json]: maybe run npm audit fix and see what it says.

**Comment by benyamin-noori** [comment]
> > I see we have security alerts on this repo that can be fixed by bumping the versions of a few dependencies.
> > Do you want to update the package lock here as well? Or would you rather open a separate PR? I'm fine with both!
> 
> Should I delete the package-lock and run npm audit? This is my first time actually fixing a situation like this.

So if you click on the security alert it tells you what versions of those packages you need to resolve this issue. You can start by deleting `package-lock.json` and `node_modules/` and running `npm i` and  seeing if it automatically bumps those versions. 

If that doesn't solve it, you need to figure out which dependencies are bringing those packages in, and bump their versions so they pick up a later version of their dependencies. 

Message me on Slack if you have questions.

**Comment by jwong-d2l** [comment]
Other than my comments above, this looks really good!

File: mixins/d2l-load-more.js [hunk 3]
```diff
@@ -29,26 +30,31 @@ const LoadMoreExemptionsBehaviorImpl = {
 		},
 		userData: {
 			type: Array,
-			value: []
+			value: [],
+			notify: true,
+			reflectToAttribute: true
```
Comment (jwong-d2l) [inline mixins/d2l-load-more.js]: do you need this?  or was it just for debugging?

**Pull request merged**